cross/hoverkite-firmware/src/protocol.rs

```diff
@@ -193,6 +193,14 @@ pub fn handle_command(
                 hoverboard.leds.green.set_low().unwrap()
             }
         }
+        Command::SetBuzzerFrequency(frequency) => {
+            log!(hoverboard.response_tx(), "Buzzer {} Hz", frequency);
+            hoverboard.buzzer.set_frequency(if frequency == 0 {
```

Comment:
I'd prefer for this command to have a duration argument and set a timer to interrupt to disable to buzzer after that time expires. This way there's less chance of accidentally setting off a forever-alarm.

`Beep { frequency, duration }`?

Reply:
Added `Command::AddBuzzerNote`.